AppendBlobReader/src/consumer.js

```diff
@@ -503,18 +354,9 @@ function setAppendBlobBatchSize(serviceBusTask) {
     } else if (serviceBusTask.containerName === "onboard-prod-applogs") {
         batchSize = 30 * 1024 * 1024;
     }
-
     return batchSize;
 }
 
-function getSumoEndpoint(serviceBusTask) {
```

Comment:
why deleting this ?

Reply:
I directly implemented the default value per the current method.
Reverting to old as suggested.